atomicapp/nulecule/main.py

```diff
@@ -102,6 +103,40 @@ def __init__(self, app_spec, destination=None,
         self.answers_file = answers_file
         self._process_answers()
 
+    @staticmethod
```

Comment:
@rtnpro - I don't believe this has been done. Can we add comments and break the function up?